.editorconfig

```diff
@@ -0,0 +1,9 @@
+[*]
+end_of_line              = crlf
+charset                  = utf-8
+indent_style             = tab
+indent_size              = tab
+trim_trailing_whitespace = true
+[*.yml]
+indent_style = space
+indent_size  = 2
```

Comment:
Do we really need an .editorconfig in the Example? I think imposing code style on end users is a bad idea.